File: Example/CoreData/SyncKit/QSEmployee+CoreDataProperties.m
//
//  QSEmployee+CoreDataProperties.m
//  SyncKitCoreDataExample
//
//  Created by Manuel Entrena on 11/01/2018.
//  Copyright © 2018 Manuel. All rights reserved.
//
//

#import "QSEmployee+CoreDataProperties.h"

@implementation QSEmployee (CoreDataProperties)

+ (NSFetchRequest<QSEmployee *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"QSEmployee"];
}

@dynamic identifier;
@dynamic name;
@dynamic photo;
@dynamic sortIndex;
@dynamic company;

@end

File: Example/CoreData/SyncKit/QSEmployee+CoreDataProperties.swift
//
//  QSEmployee+CoreDataProperties.swift
//  
//
//  Created by Jérôme Haegeli on 28.07.18.
//
//

import Foundation
import CoreData


extension QSEmployee {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<QSEmployee> {
        return NSFetchRequest<QSEmployee>(entityName: "QSEmployee")
    }

    @NSManaged public var identifier: String?
    @NSManaged public var name: String?
    @NSManaged public var photo: NSData?
    @NSManaged public var sortIndex: NSNumber?
    @NSManaged public var company: QSCompany?

}
